### PR TITLE
Created tweet page and tweet controller

### PR DESCRIPTION
**Pull request by gvanburen**

### CODE REVIEW

File: public/components/tweets/tweets.view.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+            <li><a ng-click="scrollTo('intro')">About</a></li>
+            <li><a ng-click="scrollTo('analyze')">Analyze</a></li>
+        </ul>
+    </div>
+</nav>
+<div class="heading">
+	<div class="container">
+		<div class="row">
+			<div class="col s12 m10 offset-m1">
```
Comment (TNowalk) [inline public/components/tweets/tweets.view.html]: Spacing should be consistent, either 2 or 4 spaces for your indentation.  You and Asa should agree on which amount.  The nav above uses 4 spaces, but this block here all appears to be using 2 tabs which causes some display differences when looking at the code.  My suggestion for HTML and JavaScript markup is to strictly use 2 spaces, not tabs.

File: public/components/tweets/tweets.view.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+			</div>
+		</div>
+
+	</div>
+</div>
+<div class="tweets">
+	<div class="container">
+		<div class="row">
+			<div class="col s12 m10 offset-m1">
+			<ul class="collection">
```
Comment (TNowalk) [inline public/components/tweets/tweets.view.html]: Need indentation here

File: public/components/tweets/tweets.view.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+			<li class="collection-item avatar" ng-repeat="tweet in tweets" ng-click="analyze(tweet)">
```
Comment (TNowalk) [inline public/components/tweets/tweets.view.html]: You are calling a function that is currently commented out here.  Angular is smart enough not to throw an error here, but it's a good thing to be aware of.

File: public/components/tweets/tweets.view.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+			<img src="../assets/images/bird_chart.jpeg" alt="" class="circle">
+			<span class="title"><b>{{ tweet }}</b></span>
```
Comment (TNowalk) [inline public/components/tweets/tweets.view.html]: The `<b>` tag here bothers me.  I would forgo that tag and just add a `font-style: bold;` to the `title` class.  That will eliminate unneeded markup, not to mention that the `<b>` should only be used in rare occasions.

File: public/components/tweets/tweets.view.html (hunk 1 above)
Comment (gvanburen) [inline public/components/tweets/tweets.view.html]: Just double checked - looks like we are both using 4 spaces for all files.

Comment (TNowalk) [inline public/components/tweets/tweets.view.html]: Interesting, spacing shows up funky on my side.

Comment (gvanburen) [inline public/components/tweets/tweets.view.html]: Try taking a look now. I re-did all the indentations from scratch.

**Pull request merged**